src/reducetensor.cpp

```diff
@@ -552,6 +554,11 @@ void ReduceTensorDescriptor::ReduceTensor(const Handle& handle,
     // to remove the warning from clang-tidy checking
     param += " -DMIOPEN_USE_FP32=0 -DMIOPEN_USE_FP16=0 ";
 
+#if WORKAROUND_MIOPEN_ISSUE_557
+    if(StartsWith(handle.GetDeviceName(), "gfx10"))
```

Comment:
Compilation fails. Please add `#include <miopen/stringutils.hpp>`

Reply:
Added.   Strange!  Probably because the "develop" branch I am basing my test branch on is less latest,  and which implicitly has the stringutils.hpp included through its tree of headers

Reply:
🤷‍♂️